src/frontend/web_application/src/components/ContactProfile/index.spec.js

```diff
@@ -1,7 +1,9 @@
 // import contactModule from '../index.js';
 
 describe('component contact-card-summary', () => {
-  it('needs to be implemented', () => {});
+  it('needs to be implemented', () => {
```

Comment:
It tells which tests were skipped as well:

![screenshot from 2016-12-20 10-34-24](https://cloud.githubusercontent.com/assets/245501/21345295/f4ea98f2-c69f-11e6-981e-56d931f93c40.png)